File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/AlertaDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de las alertas para su devolucion
 * 
 * @author Flor
 *
 */
public class AlertaDTO {

	private String id;
	private String idTipoAlerta;
	private TipoAlertaDTO tipoAlerta;
	private String matricula;
	private String nombreCliente;
	private String tlfContacto;
	private int vencimiento;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getIdTipoAlerta() {
		return idTipoAlerta;
	}

	public void setIdTipoAlerta(String idTipoAlerta) {
		this.idTipoAlerta = idTipoAlerta;
	}

	public TipoAlertaDTO getTipoAlerta() {
		return tipoAlerta;
	}

	public void setTipoAlerta(TipoAlertaDTO tipoAlerta) {
		this.tipoAlerta = tipoAlerta;
	}

	public String getMatricula() {
		return matricula;
	}

	public void setMatricula(String matricula) {
		this.matricula = matricula;
	}

	public String getNombreCliente() {
		return nombreCliente;
	}

	public void setNombreCliente(String nombreCliente) {
		this.nombreCliente = nombreCliente;
	}

	public String getTlfContacto() {
		return tlfContacto;
	}

	public void setTlfContacto(String tlfContacto) {
		this.tlfContacto = tlfContacto;
	}

	public int getVencimiento() {
		return vencimiento;
	}

	public void setVencimiento(int vencimiento) {
		this.vencimiento = vencimiento;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/AlertaDTO.cs

namespace Flotix2021.ModelDTO
{
    class AlertaDTO
    {
		private string _id;

		public string id
		{
			get { return _id; }
			set { _id = value; }
		}

		private string _idTipoAlerta;

		public string idTipoAlerta
		{
			get { return _idTipoAlerta; }
			set { _idTipoAlerta = value; }
		}

		private TipoAlertaDTO _tipoAlerta;

		public TipoAlertaDTO tipoAlerta
		{
			get { return _tipoAlerta; }
			set { _tipoAlerta = value; }
		}

		private string _matricula;

		public string matricula
		{
			get { return _matricula; }
			set { _matricula = value; }
		}

		private string _nombreCliente;

		public string nombreCliente
		{
			get { return _nombreCliente; }
			set { _nombreCliente = value; }
		}

		private string _tlfContacto;

		public string tlfContacto
		{
			get { return _tlfContacto; }
			set { _tlfContacto = value; }
		}

		private int _vencimiento;

		public int vencimiento
		{
			get { return _vencimiento; }
			set { _vencimiento = value; }
		}
	}
}
